qiskit/visualization/matplotlib.py

```diff
@@ -165,8 +161,6 @@ def __init__(
         # and colors 'fc', 'ec', 'lc', 'sc', 'gt', and 'tc'
         self._data = {}
         self._layer_widths = []
-        self._q_anchors = {}
```

Comment:
unclear why `_q_anchors` and `_c_anchors` are being removed here? I see they haven't been removed elsewhere, just here

Reply:
These were just moved up above at line 113.

Reply:
ah got it 😅